cli/cmd/publish_test.go

```diff
@@ -265,6 +266,12 @@ func checkCreateDocsRequest(t *testing.T, r *http.Request) {
 				"title":            "Configuration",
 				"ordinal_position": float64(2),
 			},
+			map[string]any{
+				"content":          customDocContent,
+				"name":             "custom-doc",
```

Comment:
Is this slug expected to come from the name of the doc, or from the slug in the frontmatter? Maybe we should make the `slug` in the frontmatter differ to make this clear?

Reply:
Updated the tests a bit, good point

Reply:
Great, thanks :) I think the `slug` field in the frontmatter is not being used, should we remove it?